src/setupMocks.js

```diff
@@ -0,0 +1,9 @@
+// Contains mocks which are used before any test runs
+// todo: We don't want to have these mocks. So this will be fixed in WEBAPP-170
```

Comment:
you should also move the rest of the models used in setupTest to setupMocks then

Reply:
setupTest mocks will be gone with WEBAPP-170 :) #79

Reply:
yeah but why create  an extra class setupMocks then?

Reply:
Because I want to import the mocks from somewhere else without running the code in setupTest.js

Reply:
ok sorry my bad